docs/docsite/rst/administration/tipsandtricks.rst

```diff
@@ -167,6 +167,22 @@ Using execution environments
 See :ref:`ug_execution_environments` in the |atu|.
 
 
+Adding host entries in execution environments
+===============================================
+
+In order to override DNS/host entries, add the following entries in the in the ``/etc/hosts`` file of the pod spec via the ``hostAliases`` attribute:
```

Comment:
```suggestion
In order to override DNS/host entries, add the following entries in the ``/etc/hosts`` file of the pod spec with the ``hostAliases`` attribute:
```

Reply:
Keep in mind I know zero about all of this, but I got the impression from the issue that the `hostAliases` entry was not in `/etc/hosts` but some podman file. I don't know if this meands hand-editing the `ContainerFile` produced by `ansible-builder` or some other file...

Reply:
...okay a bit of poking in this troubleshooting page and I now think the entries go in the AWX `spec` file.. But again... I'm doing major guessing here.